packages/@monocdk-experiment/assert/package.json

```diff
@@ -47,7 +47,7 @@
     "@aws-cdk/cloudformation-diff": "0.0.0"
   },
   "peerDependencies": {
-    "constructs": "^3.0.4",
+    "constructs": "^3.2.0",
```

Comment:
Should we have a linter check too? Or is this managed by dependabot and was missed?

Reply:
Looks like we explicitly don't auto-upgrade constructs - https://github.com/aws/aws-cdk/blob/4553d4b37cc7c2062eebc36bfe7a03b0226b598d/.github/workflows/yarn-upgrade.yml#L48

Linter check probably so we don't screw up when we upgrade again.

Reply:
Note about adding the linter check. We use a different version of constructs for v2, so do take that into account.

You'll need to use a different version depending on the MV. something like - https://github.com/aws/aws-cdk/blob/4553d4b37cc7c2062eebc36bfe7a03b0226b598d/tools/pkglint/lib/rules.ts#L1531